packages/medusa/src/api/routes/admin/orders/cancel-fulfillment.ts

```diff
@@ -87,3 +104,23 @@ export default async (req, res) => {
 
   res.json({ order })
 }
+
+export const adjustInventoryForCancelledFulfillment = async (
+  fulfillment: Fulfillment,
+  context: {
+    productVariantInventoryService: ProductVariantInventoryService
+  }
+) => {
+  const { productVariantInventoryService } = context
+  await Promise.all(
+    fulfillment.items.map(async ({ item, quantity }) => {
```

Comment:
suggestion(non blocking): create a variable with the items filtered before using it here.